src/Handlers/WebhookHandler.php

```diff
@@ -348,6 +347,12 @@ protected function commandPrefixes(): Collection
             ->unique();
     }
 
+    protected function createChat(Chat $telegramChat, TelegraphChat $chat): void
+    {
+        $chat->name = $this->chatName($telegramChat);
```

Comment:
Methods should generally start with an action verb like get, set, create, delete, move, etc. This helps distinguish methods from properties and makes the code more readable.

Reply:
In general, the prefix of method names is not always used. It is not needed in those classes where either there is no assignment and/or reading, or these methods are obvious by their names. In such cases, prefixes are redundant.

In this case, the code is self-documenting and it is immediately clear that we will get the chat name at the output. But, looking at the handler class again, I agree that we should add a prefix only because this approach is inherent in the class creation and a method without an action prefix is out of it.